<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <layout class="QHBoxLayout" name="seleLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="getSelectionButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>105</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Get from (sele)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ElidedLabel" name="resInfoLabel">
     <property name="minimumSize">
      <size>
       <width>161</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Residues: 0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ElidedLabel" name="atomsInfoLabel">
     <property name="minimumSize">
      <size>
       <width>161</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Atoms: 0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ElidedLabel</class>
   <extends>QLabel</extends>
   <header>widgets.elided_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
